source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+   :start-after: // start-range-search
+   :end-before: // end-range-search
+   :language: csharp
+   :dedent:
+
+The search returns the following results:
+
+.. code-block:: json
+
+   { "_id" : 3, "make" : "PRS", "description" : "High-end guitars known for their quality.", "establishedYear" : 1985, "in-stock" : true, "rating" : 9 }
+   { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in-stock" : false, "rating" : null }
+   { "_id" : 6, "make" : "Strandberg", "description" : "Modern guitars known for their headless models.", "establishedYear" : 1982, "in-stock" : false, "rating" : null }
+
+To learn more about the ``range`` operation, see the :atlas:`Atlas
+range </atlas-search/range>` guide.
+
+Regex
+~~~~~
+
+Use the ``Regex`` method to search for documents using a regular expression.
+
+The following example searches the ``guitars`` collection for documents in which
+the value of the ``make`` field contains exactly six letters.
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-regex-search
+   :end-before: // end-regex-search
+   :language: csharp
+   :dedent:
+
+The search returns the following results:
+
+.. code-block:: json
+
+   { "_id" : 1, "make" : "Fender", "description" : "Classic guitars known for their versatility.", "establishedYear" : 1946, "in-stock" : true, "rating" : 9 }
+   { "_id" : 2, "make" : "Gibson", "description" : "Classic guitars known for their rich, full tones.", "establishedYear" : 1902, "in-stock" : true, "rating" : 8 }
+   { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in-stock" : false, "rating" : null }
+   { "_id" : 5, "make" : "Ibanez", "description" : "Well-crafted guitars used by many professional guitarists.", "establishedYear" : 1957, "in-stock" : true, "rating" : 7 }
+
+.. note::
+
+   By default the ``regex`` operator cannot run on an analyzed field. You can
+   allow it to run on an analyzed field by setting ``allowAnalyzedField`` option
+   to true, as follows:
+
+   .. code-block:: csharp
+
+      var result = guitarsCollection.Aggregate()
+       .Search(Builders<Guitar>.Search.Regex(r => r.Make, regex, true))
+       .ToList();
+
+   Setting the ``allowAnalyzedField`` option to true may lead to unexpected
+   search results. To learn more, see :atlas:`regex Behavior
+   </atlas-search/regex/#behavior>`.
+
+To learn more about the ``regex`` operation, see the :atlas:`Atlas
+regex </atlas-search/regex>` guide.
+
+Span
+~~~~
+
+Use the ``Span`` method to search for text search matches within regions of a
+field. You can use this method to find strings which are near each other to
+specified degrees of precision.
+
+.. note::
+
+   The ``span`` operator is more computationally intensive than other operators,
+   because queries must keep track of positional information.
```

Comment:
Suggestion

Remove the comma after "operators" to tie the ideas closer together.